cmake-proxies/cmake-modules/AudacityTesting.cmake

```diff
@@ -9,7 +9,12 @@ if( ${_OPT}has_tests )
    enable_testing()
 
    #[[
-      add_unit_test(NAME name SOURCES file1 ... LIBRARIES lib1 ...)
+      add_unit_test(NAME name [MOCK_PREFS] [MOCK_AUDIO] SOURCES file1 ... LIBRARIES lib1 ...)
+
+      If MOCK_PREFS is specified, a test can instantiate a mocked Prefs object.
+      If MOCK_AUDIO is specified, a test will initialize PortAudio.
+
+      Audio mocking is a subject to change when Audio I/O is refactored.
```

Comment:
Is this refactoring written or just contemplated?

Reply:
Does it matter for this review?

Reply:
It is literally the next commit on the branch

Reply:
There was not, in fact, a refactoring of AudioIO, which I feared would add even more burden to this long review.

Not to imply that there are not still many virtuous refactorings to be done there some day.